app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split(" ")
```

Comment:
By default split() function split by space, so no reason to write it
```suggestion
    command_split = command.split()
```